include/rc-fshare/rtp.hpp

```diff
@@ -56,20 +56,39 @@ struct ControlMessage {
      * is to avoid loss of precision when sending float velocity values across
      * the air as ints.
      */
+    static constexpr auto POSE_SCALE_FACTOR = 1000;
     static constexpr auto VELOCITY_SCALE_FACTOR = 1000;
+    static constexpr auto ACCELERATION_SCALE_FACTOR = 500;
+
+    // Values from vision
+    int16_t vision_pose_x;
+    int16_t vision_pose_y;
+    int16_t vision_pose_theta;
+
+    // Goals
+    int16_t goal_pose_x;
+    int16_t goal_pose_y;
+    int16_t goal_pose_theta;
+
+    int16_t goal_velocity_x;
+    int16_t goal_velocity_y;
+    int16_t goal_velocity_theta;
+
+    int16_t goal_acceleration_x;
+    int16_t goal_acceleration_y;
+    int16_t goal_acceleration_theta;
 
-    //    uint8_t uid;
-    int16_t bodyX;
-    int16_t bodyY;
-    int16_t bodyW;
     int8_t dribbler;
     uint8_t kickStrength;
+
+    unsigned motionMode : 2;   // 0 = position control, 1 = velocity control
```

Comment:
Is this supposed to be 2 bits?

Reply:
Yeah. I can think of a few more modes we might want in the future, so it seems like it makes sense to leave a bit of extra room considering there's space left in the last byte of RTP packet anyway.